### ISSUE TITLE
Add event onFocusOut for input

### ISSUE THREAD

**Issue by artjerom** [vitalybaev/react-dadata#2]
If an address is selected and we clear the field, then no event occurs. Please correct

**Comment by vitalybaev** [comment]
Do you mean, that when we clear input value we should fire `onChange` event with empty address?

**Comment by iloveip** [comment]
I think when the input is cleared, the value should also be cleared. Now it stays (as you can see [here](https://codesandbox.io/embed/react-final-from-wreact-dadata-u0skz?fontsize=14)).

**Comment by vitalybaev** [comment]
Hello, sorry for the late answer.
I've published new beta version `2.0.0-beta.1`, which contains new property, `inputProps` - it contains all common properties of input tag and passes it to the input in `react-dadata`.

It allows you to pass your own `onChange` callback and make some logic:

```javascript
import { AddressSuggestions } from 'react-dadata';
import 'react-dadata/dist/react-dadata.css';

const [value, setValue] = useState();

const handleInputChange = (event) => {
  // Paste your logic here
}

<AddressSuggestions
  token="API_KEY"
  value={value}
  onChange={setValue}
  inputProps={{
    onChange: handleInputChange,
  }}
/>
```

Feel free to ask questions you have! Thanks
